expression/aggregation/base_func_test.go

```diff
@@ -39,3 +39,14 @@ func (s *testBaseFuncSuite) TestClone(c *check.C) {
 	c.Assert(desc.Args[0], check.Equals, col)
 	c.Assert(desc.equal(s.ctx, cloned), check.IsFalse)
 }
+
+func (s *testBaseFuncSuite) TestMaxMin(c *check.C) {
```

Comment:
It would be better to add the case mentioned in the issue as an integration test.
`expression/integration_test.go`